CMakeLists.txt

```diff
@@ -3,14 +3,15 @@ cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 option(MBGL_WITH_CORE_ONLY "Build only the core bits, no platform code" OFF)
 option(MBGL_WITH_COVERAGE "Enable code coverage collection" OFF)
 option(MBGL_WITH_QT "Build Mapbox GL Qt bindings" OFF)
+option(MBGL_QT_STATIC "Build Mapbox GL Qt bindings staticly" OFF)
 option(MBGL_WITH_SANITIZER "Use [address|thread|undefined] here" OFF)
 option(MBGL_WITH_RTTI "Compile with runtime type information" OFF)
 option(MBGL_WITH_OPENGL "Build with OpenGL renderer" ON)
 option(MBGL_WITH_EGL "Build with EGL renderer" OFF)
 option(MBGL_WITH_WERROR "Make all compilation warnings errors" ON)
 
 if (MBGL_WITH_QT)
-    set(CMAKE_OSX_DEPLOYMENT_TARGET 10.13)
+    set(CMAKE_OSX_DEPLOYMENT_TARGET 13.0)
```

Comment:
This sounds a bit too new. Why do you need to change it?

Reply:
Because 13.0 is now Qt minimum. It can be taht it's min for Qt 6 only. I guess a if/else could be places here?

Reply:
I guess this will not work for macOS as it's still on 12.1?

Reply:
Hmm... no idea. My main target is Qt 6. I guess the easiest would be to just to check Qt version here.

Reply:
Reverted this change and moved created override set(...) into qt.cmake where Qt version is already known.

Reply:
This still looks like 13.0 to me.

Reply:
Not sure what happened here, but we can fix it in a follow-up.

Reply:
It is already fixed by the [latter commit](https://github.com/maplibre/maplibre-gl-native/pull/197/commits/7391fc2b3ad4a16d873f4589d3e1d3170179f782) in this MR.